source/_posts/2015-09-28-the-folly-of-fatherhood.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "The Folly of Fatherhood"
+date: 2015-09-28 21:21:48 -0500
+comments: true
+categories: 
+---
+Upon reflection, I think I have observed something about fatherhood. Or at
+least _my_ fatherhood (the one I am doing or having right now, not the one my
+dad did or had with, on, at or to me—the prepositions, here, are unclear to
+me). It is this: I am fairly certain I would _destroy myself_ saving one my my
+children. In fact, I’m pretty sure I’d destroy myself _failing_ such an
```

Comment:
No. In an attempt to save my child's life that ultimately fails or is somehow obviously doomed to fail, I would still destroy myself trying. Suggestions on how to make this clearer?